WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -1729,6 +1735,7 @@ private enum ActionSheetStrings {
     static let viewProduct = NSLocalizedString("View Product in Store",
                                                comment: "Button title View product in store in Edit Product More Options Action Sheet")
     static let share = NSLocalizedString("Share", comment: "Button title Share in Edit Product More Options Action Sheet")
+    static let promoteWithBlaze = NSLocalizedString("Promote with Blaze", comment: "Button title Share in Edit Product More Options Action Sheet")
```

Comment:
Nit: I think the comment needs to be updated.

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/9868/commits/dc76e4c2a64ae852f30b4fd4f41e0b7fa3c55c98